ajax/libs/icheck-bootstrap/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "icheck-bootstrap",
+  "description": "Pure css checkboxes and radio buttons for Twitter Bootstrap.",
+  "filename": "icheck-bootstrap.css",
+  "homepage": "https://github.com/bantikyan/icheck-bootstrap#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bantikyan/icheck-bootstrap.git"
+  },
+  "author": {
+    "name": "Hovhannes Bantikyan"
+  },
+  "keywords": [
+    "checkbox",
+    "radio",
+    "bootstrap",
+    "pure",
+    "css"
+  ],
+  "license": "MIT",
+  "npmName": "icheck-bootstrap",
+  "npmFileMap": [
+    {
+      "basePath": "/",
```

```suggestion
      "basePath": "",
```

Base path should be blank for the root, not `/`